src/input_map.rs

```diff
@@ -312,11 +313,15 @@ impl<A: Actionlike> InputMap<A> {
             let mut inputs = Vec::new();
 
             for input in self.get(action.clone()).iter() {
+                let value = input_streams.get_input_value(input);
+                let axis_pair = input_streams.get_input_axis_pair(input);
                 if input_streams.input_pressed(input) {
                     inputs.push(input.clone());
-                    action_data[action.index()]
-                        .reasons_pressed
-                        .push(input.clone());
+                    let action = &mut action_data[action.index()];
+                    action.reasons_pressed.push(input.clone());
+                    action.value += value;
```

Comment:
Interesting, so then you can handle multiple overlapping inputs simultaneously 🤔 Very nice.